packages/chips/README.md

```diff
@@ -43,14 +43,19 @@ class MyApp extends Component {
 
 Prop Name | Type | Description
 --- | --- | ---
-className | String | Classes to be applied to the chip set element.
-labels | Array | An array of strings. Each string has a corresponding chip whose label will be set to the value of that string.
+className | String | Classes to be applied to the chip set element
+filter | Boolean | Indicates that the chips in the set are filter chips, which allow multiple selection from a set of options
+
 
 ### Chip
 
 Prop Name | Type | Description
 --- | --- | ---
-className | String | Classes to be applied to the chip element.
+className | String | Classes to be applied to the chip element
+id | Number | Unique identifier for the chip
```

Comment:
This would cause issues in the case of input chips because chips would re-render as indices are reassigned.

For example, we have Chip A with index 0 and Chip B with index 1. As a result of removing Chip A, Chip B's index is reassigned to 0. This would cause Chip B to re-render, which would trigger the entry animation, even though nothing has happened to Chip B.

Reply:
Looking forward to entry chips, if we wanted to create a new chip, what guidance will we provide? How will the id be generated?

Reply:
Still need to experiment with this more, but we'll likely provide guidance for the user write most of the logic for adding a input chip. The `Chip` will handle all animations. The user will generate the id's since they'll be managing selected chips using their id's.